.github/workflows/build_and_test.yml

```diff
@@ -90,6 +90,7 @@ jobs:
         id: build-tag
 
   build-buildtools-image:
+    if: github.event.action != 'ready_for_review'
```

Comment:
I'm not sure if we need to have this condition, am I missing something?

Reply:
build-buildtools-image job rerun when our PR is moved to 'Ready for review'.  Hence to avoid that need to add this check

Reply:
> build-buildtools-image job rerun when our PR is moved to 'Ready for review'

Hmm, this is unexpected, the workflow is not triggered on status change (`draft` -> `ready_for_review`). Maybe this is outdated logic from the previous attempt (when the workflow got triggered by `ready_for_review`?)

`github.event.action != 'ready_for_review'` is always true, workflow handles only `opened`, `synchronize`, and `reopened` types afaics

Reply:
ahh yep, absolutely spot on. literally skipped that out of my mind.  the workflow is not triggered by ready for review and yes this is unnecessary.